.classpath

```diff
@@ -1,51 +1,51 @@
-	<classpathentry kind="lib" path="lib/commons-codec-1.9.jar"/>
-	<classpathentry kind="lib" path="lib/commons-collections-3.2.2.jar"/>
-	<classpathentry kind="lib" path="lib/commons-configuration-1.9.jar"/>
-	<classpathentry kind="lib" path="lib/commons-httpclient-3.1.jar"/>
-	<classpathentry kind="lib" path="lib/commons-io-2.4.jar"/>
-	<classpathentry kind="lib" path="lib/commons-lang-2.6.jar"/>
-	<classpathentry kind="lib" path="lib/diffutils-1.2.1.jar"/>
-	<classpathentry kind="lib" path="lib/dom4j-1.6.1.jar"/>
-	<classpathentry kind="lib" path="lib/flex-messaging-common.jar"/>
-	<classpathentry kind="lib" path="lib/flex-messaging-core.jar"/>
-	<classpathentry kind="lib" path="lib/groovy-2.0.4.jar"/>
-	<classpathentry kind="lib" path="lib/groovy-xml-2.0.4.jar"/>
-	<classpathentry kind="lib" path="lib/hsqldb-2.3.3.jar"/>
-	<classpathentry kind="lib" path="lib/jdom.jar"/>
-	<classpathentry kind="lib" path="lib/jericho-html-3.1.jar"/>
-	<classpathentry kind="lib" path="lib/jfxrt.jar"/>
-	<classpathentry kind="lib" path="lib/jgraphx-2.0.0.1.jar"/>
-	<classpathentry kind="lib" path="lib/joda-time-2.2.jar"/>
-	<classpathentry kind="lib" path="lib/json-lib-2.4-jdk15.jar"/>
-	<classpathentry kind="lib" path="lib/json-simple-1.1.1.jar"/>
-	<classpathentry kind="lib" path="lib/jsoup-1.7.2.jar"/>
-	<classpathentry kind="lib" path="lib/log4j-1.2.16.jar"/>
-	<classpathentry kind="lib" path="lib/org.eclipse.birt.runtime_4.2.2.v20130216-1152.jar"/>
-	<classpathentry kind="lib" path="lib/org.jwall.web.audit-0.2.15.jar"/>
-	<classpathentry kind="lib" path="lib/rsyntaxtextarea-2.5.8.jar"/>
-	<classpathentry kind="lib" path="lib/soa-model-core-1.4.1.4.jar"/>
-	<classpathentry kind="lib" path="lib/swingx-all-1.6.4.jar"/>
-	<classpathentry kind="lib" path="lib/ZAPSNITerminator.jar"/>
-	<classpathentry kind="lib" path="testlib/cglib-nodep-2.2.jar"/>
-	<classpathentry kind="lib" path="testlib/hamcrest-all-1.3.jar"/>
-	<classpathentry kind="lib" path="testlib/javassist-3.16.1-GA.jar"/>
-	<classpathentry kind="lib" path="testlib/junit-dep-4.11.jar"/>
-	<classpathentry kind="lib" path="testlib/mockito-all-1.9.0.jar"/>
-	<classpathentry kind="lib" path="testlib/objenesis-1.2.jar"/>
-	<classpathentry kind="lib" path="testlib/powermock-mockito-1.4.12-full.jar"/>
-	<classpathentry kind="lib" path="testlib/simple-5.0.2.jar"/>
-	<classpathentry kind="lib" path="lib/selenium-release-3.zap"/>
-	<classpathentry kind="lib" path="lib/zap-2.5.0.jar"/>
-	<classpathentry kind="lib" path="lib/json-20160212.jar"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
-	<classpathentry kind="lib" path="lib/metadata-extractor-2.8.1.jar"/>
-	<classpathentry kind="lib" path="lib/xmpcore-5.1.2.jar"/>
-	<classpathentry kind="lib" path="lib/github-api-1.75.jar"/>
-	<classpathentry kind="lib" path="lib/j2bugzilla-2.2.1.jar"/>
-	<classpathentry kind="output" path="bin"/>
-</classpath>
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
The changes to this file can be reverted(?), also, the branch could be rebased to include the latest zap.jar (from #799).